app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityPresenter.kt

```diff
@@ -1,37 +1,21 @@
 package org.oppia.android.app.settings.profile
 
-import android.content.Intent
-import android.view.KeyEvent
-import android.view.inputmethod.EditorInfo
 import androidx.appcompat.app.AppCompatActivity
 import androidx.databinding.DataBindingUtil
-import androidx.lifecycle.Observer
 import org.oppia.android.R
 import org.oppia.android.app.activity.ActivityScope
-import org.oppia.android.app.model.ProfileId
-import org.oppia.android.app.translation.AppLanguageResourceHandler
-import org.oppia.android.app.utility.TextInputEditTextHelper.Companion.onTextChanged
-import org.oppia.android.app.viewmodel.ViewModelProvider
 import org.oppia.android.databinding.ProfileResetPinActivityBinding
 import org.oppia.android.domain.profile.ProfileManagementController
-import org.oppia.android.util.data.DataProviders.Companion.toLiveData
 import javax.inject.Inject
 
 /** The presenter for [ProfileResetPinActivity]. */
 @ActivityScope
 class ProfileResetPinActivityPresenter @Inject constructor(
   private val activity: AppCompatActivity,
   private val profileManagementController: ProfileManagementController,
```

Comment:
Also this is unused now and therefore can be removed.

Reply:
done